app/screens/channel/channel_nav_bar/channel_nav_bar.js

```diff
@@ -62,11 +62,13 @@ export default class ChannelNavBar extends PureComponent {
         const padding = {paddingHorizontal: 0};
 
         let height;
+        let canHaveSubtitle = true;
         switch (Platform.OS) {
         case 'android':
             height = ANDROID_TOP_PORTRAIT;
             if (DeviceTypes.IS_TABLET) {
                 height = ANDROID_TOP_LANDSCAPE;
+                canHaveSubtitle = false;
```

Comment:
why the subtitle is not shown on a tablet in landscape mode? it should have enough real state to show the subtitle

Reply:
Yes, you are right, there is enough space. removing the `false` here

Reply:
Done